### PR TITLE
[aws-rds] Make it possible to get normal value from all engine.

### PR DESCRIPTION
**Pull request by sioncojp**
### reference
Reference to below. But any metrics are not in the page, so I got all of the data in real situations.
http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Aurora.Monitoring.html
http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html

### before output
```shell
### aurora
2017/01/20 17:14:54 BinLogDiskUsage: fetched no datapoints
2017/01/20 17:14:54 CPUCreditUsage: fetched no datapoints
2017/01/20 17:14:54 ReplicaLag: fetched no datapoints
2017/01/20 17:14:54 CPUCreditBalance: fetched no datapoints
rds.DiskQueueDepth.DiskQueueDepth	0.096872	1484900094
rds.FreeableMemory.FreeableMemory	625827840	1484900094
rds.SwapUsage.SwapUsage	0	1484900094
rds.NetworkThroughput.NetworkTransmitThroughput	1365.262175	1484900094
rds.NetworkThroughput.NetworkReceiveThroughput	76.319211	1484900094
rds.CPUUtilization.CPUUtilization	1.020000	1484900094
rds.FreeStorageSpace.FreeStorageSpace	4454490112.000000	1484900094
rds.Latency.ReadLatency	0	1484900094
rds.Latency.WriteLatency	0.020726	1484900094
rds.MaximumUsedTransactionIDs.MaximumUsedTransactionIDs	125	1484900094
rds.ReplicationSlotDiskUsage.ReplicationSlotDiskUsage	4096	1484900094
rds.TransactionLogsDiskUsage.TransactionLogsDiskUsage	570433829	1484900094
rds.DatabaseConnections.DatabaseConnections	0	1484900094
rds.IOPS.ReadIOPS	0	1484900094
rds.IOPS.WriteIOPS	3.183439	1484900094
rds.Throughput.ReadThroughput	0	1484900094
rds.Throughput.WriteThroughput	291649.782489	1484900094
rds.OldestReplicationSlotLag.OldestReplicationSlotLag	-1	1484900094
rds.TransactionLogsGeneration.TransactionLogsGeneration	284349.423729	1484900094


### mysql
2017/01/20 17:15:21 CPUCreditBalance: fetched no datapoints
2017/01/20 17:15:21 CPUCreditUsage: fetched no datapoints
2017/01/20 17:15:21 ReplicaLag: fetched no datapoints
rds.BinLogDiskUsage.BinLogDiskUsage	999	1484900121
rds.Latency.ReadLatency	0	1484900121
rds.Latency.WriteLatency	0	1484900121
rds.FreeableMemory.FreeableMemory	572526592	1484900121
rds.FreeStorageSpace.FreeStorageSpace	4759085056.000000	1484900121
rds.NetworkThroughput.NetworkTransmitThroughput	1361.350000	1484900121
rds.NetworkThroughput.NetworkReceiveThroughput	94.050000	1484900121
rds.DatabaseConnections.DatabaseConnections	0	1484900121
rds.Throughput.ReadThroughput	0	1484900121
rds.Throughput.WriteThroughput	546.133333	1484900121
rds.DiskQueueDepth.DiskQueueDepth	0	1484900121
rds.CPUUtilization.CPUUtilization	0.830000	1484900121
rds.SwapUsage.SwapUsage	0	1484900121
rds.IOPS.ReadIOPS	0	1484900121
rds.IOPS.WriteIOPS	0.033333	1484900121


### postgresql
2017/01/20 17:15:47 DiskQueueDepth: fetched no datapoints
2017/01/20 17:15:47 FreeStorageSpace: fetched no datapoints
2017/01/20 17:15:47 SwapUsage: fetched no datapoints
2017/01/20 17:15:47 ReadLatency: fetched no datapoints
2017/01/20 17:15:47 WriteLatency: fetched no datapoints
2017/01/20 17:15:47 ReadThroughput: fetched no datapoints
2017/01/20 17:15:47 WriteThroughput: fetched no datapoints
2017/01/20 17:15:47 ReplicaLag: fetched no datapoints
2017/01/20 17:15:47 ReadIOPS: fetched no datapoints
2017/01/20 17:15:47 WriteIOPS: fetched no datapoints
rds.NetworkThroughput.NetworkTransmitThroughput	123603.920131	1484900147
rds.NetworkThroughput.NetworkReceiveThroughput	79363.962132	1484900147
rds.EngineUptime.EngineUptime	2585820	1484900147
rds.Queries.Queries	308.816423	1484900147
rds.Transaction.ActiveTransactions	0	1484900147
rds.Transaction.BlockedTransactions	0	1484900147
rds.DatabaseConnections.DatabaseConnections	115	1484900147
rds.LoginFailures.LoginFailures	0.016663	1484900147
rds.AuroraBinlogReplicaLag.AuroraBinlogReplicaLag	0	1484900147
rds.AuroraReplicaLag.AuroraReplicaLagMaximum	21.275000	1484900147
rds.AuroraReplicaLag.AuroraBinlogReplicaLag	0	1484900147
rds.CPUUtilization.CPUUtilization	9.400000	1484900147
rds.FreeableMemory.FreeableMemory	10807275520.000000	1484900147
rds.CacheHitRatio.ResultSetCacheHitRatio	5.526316	1484900147
rds.CacheHitRatio.BufferCacheHitRatio	99.760848	1484900147
rds.Deadlocks.Deadlocks	0	1484900147
rds.BinLogDiskUsage.BinLogDiskUsage	0	1484900147
```

### after output
```shell
### aurora
rds.Latency.SelectThroughput	6.886777	1484899669
rds.Latency.InsertLatency	3.039315	1484899669
rds.Latency.UpdateLatency	0.812045	1484899669
rds.Latency.DeleteLatency	0.807750	1484899669
rds.Latency.CommitLatency	3.681126	1484899669
rds.Latency.DDLLatency	0	1484899669
rds.Latency.DMLLatency	1.037085	1484899669
rds.Throughput.SelectThroughput	6.886777	1484899669
rds.Throughput.InsertThroughput	8.587627	1484899669
rds.Throughput.UpdateThroughput	76.338169	1484899669
rds.Throughput.DeleteThroughput	0.066700	1484899669
rds.Throughput.CommitThroughput	65.366016	1484899669
rds.Throughput.DDLThroughput	0	1484899669
rds.Throughput.DMLThroughput	84.992496	1484899669
rds.Queries.Queries	279.227166	1484899669
rds.CacheHitRatio.ResultSetCacheHitRatio	7.021792	1484899669
rds.CacheHitRatio.BufferCacheHitRatio	99.241267	1484899669
rds.AuroraBinlogReplicaLag.AuroraBinlogReplicaLag	0	1484899669
rds.AuroraReplicaLag.AuroraReplicaLagMaximum	7.052000	1484899669
rds.AuroraReplicaLag.AuroraBinlogReplicaLag	0	1484899669
rds.Deadlocks.Deadlocks	0	1484899669
rds.Transaction.ActiveTransactions	0	1484899669
rds.Transaction.BlockedTransactions	0	1484899669
rds.EngineUptime.EngineUptime	2585340	1484899669
rds.LoginFailures.LoginFailures	0.016670	1484899669

### mysql
2017/01/20 17:07:37 CPUCreditBalance: fetched no datapoints
2017/01/20 17:07:37 CPUCreditUsage: fetched no datapoints
rds.CPUCreditBalance.CPUCreditBalance	0	1484899657
rds.CPUCreditUsage.CPUCreditUsage	0	1484899657
rds.DatabaseConnections.DatabaseConnections	0	1484899657
rds.FreeStorageSpace.FreeStorageSpace	4759085056	1484899657
rds.Latency.ReadLatency	0	1484899657
rds.Latency.WriteLatency	0	1484899657
rds.Throughput.ReadThroughput	0	1484899657
rds.Throughput.WriteThroughput	1570.107165	1484899657
rds.NetworkThroughput.NetworkTransmitThroughput	1387.333333	1484899657
rds.NetworkThroughput.NetworkReceiveThroughput	98.383333	1484899657
rds.DiskQueueDepth.DiskQueueDepth	0	1484899657
rds.FreeableMemory.FreeableMemory	572510208	1484899657
rds.SwapUsage.SwapUsage	0	1484899657
rds.IOPS.ReadIOPS	0	1484899657
rds.IOPS.WriteIOPS	0.199997	1484899657
rds.BinLogDiskUsage.BinLogDiskUsage	697	1484899657
rds.CPUUtilization.CPUUtilization	1.020000	1484899657

### postgresql
2017/01/20 17:07:20 CPUCreditUsage: fetched no datapoints
2017/01/20 17:07:20 CPUCreditBalance: fetched no datapoints
rds.CPUUtilization.CPUUtilization	0.980000	1484899640
rds.ReplicationSlotDiskUsage.ReplicationSlotDiskUsage	4096	1484899640
rds.TransactionLogsDiskUsage.TransactionLogsDiskUsage	570433829	1484899640
rds.CPUCreditUsage.CPUCreditUsage	0	1484899640
rds.FreeableMemory.FreeableMemory	625999872	1484899640
rds.FreeStorageSpace.FreeStorageSpace	4454580224	1484899640
rds.NetworkThroughput.NetworkTransmitThroughput	1389.364751	1484899640
rds.NetworkThroughput.NetworkReceiveThroughput	84.355419	1484899640
rds.MaximumUsedTransactionIDs.MaximumUsedTransactionIDs	124	1484899640
rds.IOPS.ReadIOPS	1.099853	1484899640
rds.IOPS.WriteIOPS	0.616584	1484899640
rds.Throughput.ReadThroughput	682.575657	1484899640
rds.Throughput.WriteThroughput	6757.499000	1484899640
rds.TransactionLogsGeneration.TransactionLogsGeneration	9.333333	1484899640
rds.OldestReplicationSlotLag.OldestReplicationSlotLag	-1	1484899640
rds.DiskQueueDepth.DiskQueueDepth	0.000600	1484899640
rds.CPUCreditBalance.CPUCreditBalance	0	1484899640
rds.DatabaseConnections.DatabaseConnections	0	1484899640
rds.SwapUsage.SwapUsage	0	1484899640
rds.Latency.ReadLatency	0.000800	1484899640
rds.Latency.WriteLatency	0.000124	1484899640
```

### CODE REVIEW

File: mackerel-plugin-aws-rds/lib/metrics.go [hunk 1]
```diff
@@ -63,6 +96,32 @@ func (p RDSPlugin) auroraGraphDefinition() map[string]mp.Graphs {
...
+			Label: p.LabelPrefix + " Throughput",
+			Unit:  "bytes/sec",
+			Metrics: []mp.Metrics{
+				{Name: "SelectThroughput", Label: "Read"},
+				{Name: "InsertThroughput", Label: "Write"},
+				{Name: "UpdateThroughput", Label: "Write"},
+				{Name: "DeleteThroughput", Label: "Write"},
+				{Name: "CommitThroughput", Label: "Write"},
+				{Name: "DDLThroughput", Label: "Write"},
+				{Name: "DMLThroughput", Label: "Write"},
```
Comment (itchyny) [inline mackerel-plugin-aws-rds/lib/metrics.go]: Please specify labels to distinguish metrics from each other.

**Comment by itchyny** [comment]
The labels like OldestReplicationSlotLag seem to be kept as the same but I think more readable labels are preferable... (you can change in this pull request or create a new one, I leave it to you)

**Comment by sioncojp** [comment]
I'm gonna fix that [L100](https://github.com/mackerelio/mackerel-agent-plugins/blob/master/mackerel-plugin-aws-rds/lib/metrics.go#L100) (differece to name, label)

**Comment by sioncojp** [comment]
@itchyny 
Please give me some advice that I think some label names like:
```
{Name: "OldestReplicationSlotLag", Label: "SlotLag"},
{Name: "ReplicationSlotDiskUsage", Label: "SlotDiskUsage"},
{Name: "AuroraReplicaLagMaximum", Label: "Maximum"},
```
If you will approve, I'm gonna fix like that.

**Comment by itchyny** [comment]
That looks good to me.

**Comment by sioncojp** [comment]
Fix all. Please re-review :)

**Comment by itchyny** [comment]
LGTM!

**Pull request merged**